lib/timeutils/unixtime.c

```diff
@@ -371,7 +371,7 @@ guint64
 unix_time_to_unix_epoch(const UnixTime ut)
```

Comment:
this function name is misleading for me. I know this patch does not introduce these, but for me "time since unix epoch" is in seconds and not in any particular resolution (msec, usecs or nsecs).

I would rename these to unix_time_from_unix_epoch_usec() and unix_time_to_unix_epoch_usec() to show the 
granularity.

Later (if needed we can add the sec/nsec variants as well).

Reply:
Thanks for annotating these functions.